lisa/features/gpu.py

```diff
@@ -67,8 +67,6 @@ class Gpu(Feature):
         "libelf-dev",
         "linux-tools-$(uname -r)",
         "linux-cloud-tools-$(uname -r)",
-        "python3",
```

Comment:
Make sure no regression on existing versions.

Reply:
Checked load_gpu case for 1604, 1804, 2004, 2204, 2304, 2310